jung-visualization/src/main/java/edu/uci/ics/jung/visualization/VisualizationModel.java

```diff
@@ -20,6 +20,11 @@
 /** */
 public interface VisualizationModel<N, E, P> extends LayoutModelAware<N, E, P> {
 
+  enum ModelStructure {
```

Comment:
This feels like an awkward name.  Maybe SpatialSupport?

Reply:
yeah, every time i choose a name for something I think of GraphZoomScrollPane